### PR TITLE
docker-*: add French translation

### PR DESCRIPTION
**Pull request by titouan-joseph**
- [x] The page(s) are in the correct platform directories: `common`.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
Welcome to tldr @titouan-joseph , Thanks for your contribution ❤️. This is a very massive PR 😄 . I checked all the files using Google Translate. I have some suggestions that need to be addressed.

File: pages.fr/common/docker-ps.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- Lister les conteneurs Docker en cours d'exécution :
+
+`docker ps`
+
+- Lister tous les conteneurs Docker (en cours d'exécution et arrêtés) :
+
+`docker ps --all`
+
+- Afficher le dernier conteneur Docker créé (avec tous les infos):
```
[the remark this answers is not in the document]
Comment (titouan-joseph) [reply pages.fr/common/docker-ps.md]: Done in commit [a558bbe](https://github.com/tldr-pages/tldr/pull/8529/commits/a558bbe4e6af9495658aa2849141ef0a3829d3a6)

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks

**Review by titouan-joseph — COMMENTED** [review]
Good for me

File: pages.fr/common/docker-image.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# docker image
+
+> Gérer les images Docker.
+> Voir aussi  `docker build`, `docker import`, and `docker pull`.
```
Comment (navarroaxel) [inline pages.fr/common/docker-image.md:4]:
```suggestion
> Voir aussi `docker build`, `docker import`, and `docker pull`.
```

**Review by navarroaxel — APPROVED** [review]
Hi @titouan-joseph, welcome and thanks for your submission. I'll apply a little fix then merge this.

**Pull request merged**